src/index.html

```diff
@@ -11,7 +11,117 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form id="myForm" action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST">
+      <div class="group">
+        <fieldset>
+          <legend>Personal information</legend>
+          <label for="surname" class="label-margin">Surname:
+            <input type="text" id="surname" name="surname" 
+            autocomplete="off"
+            >
+          </label>
```

Comment:
```suggestion
          <label for="surname" class="label-margin">
            Surname:
            <input 
              type="text" 
              id="surname" 
              name="surname" 
              autocomplete="off"
            >
          </label>
```
Fix all cases
![image](https://user-images.githubusercontent.com/96409563/225106358-7e173190-3533-4570-8d24-26f2c948dbac.png)

Reply:
updated thank you